doc/user/content/transform-data/idiomatic-materialize-sql/appendix/window-function-to-materialize.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Window function to idiomatic Materialize"
+description: "Cheatsheet for window functions to idiomatic Materialize SQL."
+menu:
+  main:
+    parent: idiomatic-materialize-appendix
+    weight: 20
+    identifier: idiomatic-materialize-appendix-window-functions-to-materialize
```

Comment:
https://preview.materialize.com/materialize/29643/transform-data/idiomatic-materialize-sql/appendix/window-function-to-materialize/